src/Speech/V1/SpeechClient.php

```diff
@@ -30,13 +30,77 @@
 
 namespace Google\Cloud\Speech\V1;
 
+use Generator;
+use Google\Cloud\Speech\SpeechHelpersTrait;
 use Google\Cloud\Speech\V1\Gapic\SpeechGapicClient;
 
 /**
  * {@inheritdoc}
  */
 class SpeechClient extends SpeechGapicClient
 {
-    // This class is intentionally empty, and is intended to hold manual
-    // additions to the generated {@see SpeechClientImpl} class.
+    use SpeechHelpersTrait;
+
+    /**
+     * Creates a stream of StreamingRecognizeRequest objects from a stream of audio data.
+     *
+     * @param iterable|string[] $chunks
+     * @return Generator|StreamingRecognizeRequest[]
+     */
+    public function createRequestStream($chunks)
+    {
+        return $this->createRequestStreamHelper(StreamingRecognizeRequest::class, $chunks);
+    }
+
+    /**
+     * Performs speech recognition on a stream of audio data. This method is only available via
+     * the gRPC API (not REST).
```

Comment:
What happens if gRPC is not installed? (I see no obvious exception bubbling.)

Reply:
The exception will be thrown in the REST transport object: https://github.com/googleapis/gax-php/blob/master/src/ApiCore/Transport/RestTransport.php#L96